src/modules/admin/product/controllers/dtos/product/create-product.dtos.ts

```diff
@@ -27,6 +27,20 @@ export class CreateProductRequestDTO {
 	@IsNotEmpty()
 	product_description: string
 
+	@ApiProperty({
```

Comment:
Please check the requirements if **product_specifications** and **product_storage_instructions** is required or not